policyengine_us/parameters/gov/states/oh/tax/income/credits/retirement_income/amount.yaml

```
@@ -0,0 +1,40 @@
+description: Ohio provides this credit amount under the Retirement Income Credit.
+brackets:
+  - threshold:
+      2021-01-01: 0
+    amount:
+      2021-01-01: 0
+  - threshold: 
+      2021-01-01: 501
```

And elsewhere. You can use `calc(..., right=True)`.
```suggestion
      2021-01-01: 500
```